test/functionalTest/cases/1050_options_unique/options_unique.test

```diff
@@ -0,0 +1,190 @@
+
+--NAME--
+Get entity attributes with unique values
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0-255
+
+--SHELL--
+
+#
+# 01. Create an entity CarPool with 6 attributes with values that are car brands
+# 02. Get all values of entity CarPool
+# 03. Get all UNIQUE values of entity CarPool
+#
+# Error Handling
+# 04. URI param options with both 'keyValues' and 'values', see error
+# 05. URI param options with both 'keyValues' and 'unique', see error
+# 06. URI param options with both 'values' and 'unique', see error
+#
+
+
+echo "01. Create an entity CarPool with 6 attributes with values that are car brands"
+echo "=============================================================================="
+payload='{
+  "id": "CarPool",
+  "Car1": {
+    "value": "Mercedes"
+  },
+  "Car2": {
+    "value": "BMW"
+  },
+  "Car3": {
+    "value": "Audi"
+  },
+  "Car4": {
+    "value": "BMW"
+  },
+  "Car5": {
+    "value": "Mercedes"
+  },
+  "Car6": {
+    "value": "BMW"
+  }
+}'
+orionCurl --url /v2/entities --payload "$payload" --json
+echo
+echo
+
+
+echo "02. Get all values of entity CarPool"
+echo "===================================="
+orionCurl --url '/v2/entities/CarPool?options=values' --json
+echo
+echo
+
+
+echo "03. Get all UNIQUE values of entity CarPool"
+echo "==========================================="
+orionCurl --url '/v2/entities/CarPool?options=unique' --json
+echo
+echo
+
+
+echo "04. URI param options with both 'keyValues' and 'values', see error"
+echo "==================================================================="
+orionCurl --url '/v2/entities/CarPool?options=keyValues,values' --json
+echo
+echo
+
+
+echo "05. URI param options with both 'keyValues' and 'unique', see error"
+echo "==================================================================="
+orionCurl --url '/v2/entities/CarPool?options=keyValues,unique' --json
+echo
+echo
+
+
+echo "06. URI param options with both 'values' and 'unique', see error"
+echo "================================================================"
+orionCurl --url '/v2/entities/CarPool?options=values,unique' --json
+echo
+echo
+
+
+
+--REGEXPECT--
+01. Create an entity CarPool with 6 attributes with values that are car brands
+==============================================================================
+HTTP/1.1 201 Created
+Content-Length: 0
+Location: /v2/entities/CarPool?type=none
+Date: REGEX(.*)
+
+
+
+02. Get all values of entity CarPool
+====================================
+HTTP/1.1 200 OK
+Content-Length: 48
+Content-Type: application/json
+Date: REGEX(.*)
+
+[
+    "Mercedes",
+    "BMW",
+    "Audi",
+    "BMW",
+    "Mercedes",
+    "BMW"
+]
+
+
+03. Get all UNIQUE values of entity CarPool
+===========================================
+HTTP/1.1 200 OK
+Content-Length: 25
+Content-Type: application/json
+Date: REGEX(.*)
+
+[
+    "Mercedes",
+    "BMW",
+    "Audi"
+]
+
+
+04. URI param options with both 'keyValues' and 'values', see error
```

Comment:
Yeah, they could. However, the test file was so small any way ...

If you insist, I have no problem in moving the 'error part' to another functest ;-)